tests/api/test_algolia.py

```diff
@@ -0,0 +1,16 @@
+import requests
+
+from actions import config
+from actions.api.algolia import AlgoliaAPI
+
+
+def test_get_algolia_link():
+    """Test that the link returned is correctly formatted and goes to a valid webpage"""
+    algolia = AlgoliaAPI(
+        config.algolia_app_id, config.algolia_search_key, config.algolia_docs_index
+    )
+    algolia_result = algolia.search("rasa")
+    link_string = algolia.get_algolia_link(algolia_result.get("hits"), 0)
+    link = link_string.split("](")[1][:-1]
```

Comment:
> Test that the link returned is correctly formatted

Maybe add an assertion of a regex match before we go about splitting the link string? That'd help to catch any regression without the test failing in a weird way when you try to split it